pkg/controller/stacks/install/installjob.go

```diff
@@ -82,11 +84,15 @@ func createInstallJob(i v1alpha1.StackInstaller, executorInfo *stacks.ExecutorIn
 			BackoffLimit: &jobBackoff,
 			Template: corev1.PodTemplateSpec{
 				Spec: corev1.PodSpec{
-					RestartPolicy: corev1.RestartPolicyNever,
+					AutomountServiceAccountToken: &passServiceAccount,
```

Comment:
Chatted with @suskin and @muvaf about this, we agree that the lockdown of the `initContainers` and unpack should introduced in a separate PR.

Reply:
Yes - we aren't opposed to doing it, but moving it into a separate PR helps us reduce the reviewing and testing overhead, and the potential blast radius, by making more smaller PRs. Which is a benefit that we like about smaller PRs.